modules/distribution/resources/operation_policies/specifications/ccAddHeader_v1.json

```diff
@@ -9,15 +9,15 @@
       "name": "headerName",
       "displayName": "Header Name",
       "description": "Name of the header to be added",
-      "validationRegex": "^([a-zA-Z_][a-zA-Z\\d_\\-\\ ]*)$",
+      "validationRegex": "^([a-zA-Z_\\:][a-zA-Z\\d_\\-\\ ]*)$",
       "type": "String",
       "required": true
     },
     {
       "name": "headerValue",
       "displayName": "Header Value",
       "description": "Value of the header",
-      "validationRegex": "^([a-zA-Z\\d_][a-zA-Z\\d_\\-\\ ]*)$",
+      "validationRegex": "^.+$",
```

Comment:
Are we leaving this as is?

Reply:
yes, let's enable any value for the header value, including Unicode chars